.github/workflows/flutter_vm.yaml

```diff
@@ -0,0 +1,67 @@
+name: Flutter (VM)
+on:
+  workflow_call:
+    inputs:
+      working-directory:
+        description: The working directory relative to the repo root
+        required: true
+        type: string
+
+jobs:
+  test:
+    name: Test
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    strategy:
+      # Allows other matrix items to run if one fails
+      fail-fast: false
+      matrix:
+        sdk:
+          - stable
+          - master
+    steps:
+      - name: Git Checkout
+        uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # 3.1.0
+
+      - name: Setup Flutter
+        uses: subosito/flutter-action@dbf1fa04f4d2e52c33185153d06cdb5443aa189d # 2.7.0
```

Comment:
uber nit: isn't this 2.8.0 https://github.com/subosito/flutter-action/commit/dbf1fa04f4d2e52c33185153d06cdb5443aa189d?

Reply:
Whoops. Updated 👍